app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,9 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.SuperMegaSnake"
         tools:targetApi="31">
+        <activity
+            android:name=".SinglePlayerDefeatActivity"
```

Comment:
This activity name will conflict with the non-record single player defeat activity.